src/components/Filters/components/Count/Count.tsx

```diff
@@ -9,23 +11,34 @@ const Count: React.FC<NumberProps> = ({
   min,
   max,
   placeholder = 'Count',
+  patternMessage = 'Use only integers',
+  onError,
 }) => {
+  const [isInvalid, setIsInvalid] = useState(false);
+
   const handleKeyPress = (event) => {
     if (!/[0-9]/.test(event.key)) {
       event.preventDefault();
     }
   };
+  const handleOnChange = (event) => {
+    onChange(event);
+    const error = !validateNumber(event.target.value, min, max);
+    setIsInvalid(error);
+    onError(error);
+  };
 
   return (
-    <StyledNumber
-      max={max}
```

Comment:
We can't use `min`, `max`, `step` native attributes?

Reply:
They would be useless to add here. The range validation is done in `validateNumber` so we can show our custom errors.

Reply:
I understand how it works. I thought we agreed we will use native validation and just display custom message for sake of simplicity, so it is not possible?

Reply:
It is possible but does nothing as I understand